docs/example.md

```diff
@@ -1,24 +1,25 @@
 # Full workflow tutorial
 
-This example demonstrates how to export application deployed on Openshift,
-to Nulecule specification and then import that application to Kubernetes.
+This example demonstrates how to export application deployed on Openshift
+to Nulecule specification and then import that application to Kubernetes and/or OpenShift.
 
 This is a 3 step tutorial:
```

Comment:
Also adding a blank line after this will make the 1. 2. and 3. get onto separate lines.